test/e2e/bootstrap_resources.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
This is a very outdated version of test bootstrapping. It is no longer supported in `acktest`, so I would recommend you update it using the eks controller as your guide.